.github/workflows/build-deps.yml

```diff
@@ -0,0 +1,196 @@
+name: Build emsdk dependencies
+
+on:
+  workflow_dispatch:
+    inputs:
+      llvm_build:
+        description: 'LLVM Webassembly arm64'
+        required: false
+        default: false
+        type: boolean
+      llvm_build_cross_compilation:
+        description: 'LLVM Webassembly arm64 (cross compilation)'
+        required: false
+        default: false
+        type: boolean
+      emscripten_build:
+        description: 'Emscripten'
+        required: false
+        default: false
+        type: boolean
+      nodejs_build:
+        description: 'Node.js arm64'
+        required: false
+        default: false
+        type: boolean
+      emsdk_build:
+        description: 'emsdk arm64'
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  build-llvm-webassembly-arm64:
+    if: ${{ inputs.llvm_build }} ||  ${{ inputs.emsdk_build }}
+
+    runs-on: [self-hosted, Windows, ARM64, WASM]
+    timeout-minutes: 600
+
+    steps:
+    
+    - name: Build LLVM WebAssembly arm64
+      shell: cmd
+      run: |
+        exit
```

Comment:
What is this `exit` here for?

Reply:
it was added for test purpose to back existing bits of llvm without rebuild which takes hours.  It will be removed

Reply:
"exit" command has been removed after testing